valve-monitor/web-app/src/services/prisma-datastore/PrismaAttributeStore.ts

```diff
@@ -4,6 +4,14 @@ import { AttributeStore } from "../AttributeStore";
 import { DeviceID, FleetID } from "../DomainModel";
 import { PrismaDataProvider } from "./PrismaDataProvider";
 
+function uuidv4() {
```

Comment:
this is a very fancy function

Reply:
Yeah I think I’ll add a comment for why this needs to exist, and maybe link to the Stack Overflow post I got it from 🙃